Ideas-2021.md

```diff
@@ -96,18 +96,13 @@ Suggesting or adding features, fixing bugs, or releasing activities
 will help you to gain experience
 
 **Coding Mentors**<br>
-[Jui Pradhan](https://github.com/JuiP), [Srevin Saju](https://github.com/srevinsaju), [Saumya Mishra](https://github.com/Saumya-Mishra9129)
-
+[Jui Pradhan](https://github.com/JuiP), [Srevin Saju](https://github.com/srevinsaju), [Saumya Mishra](https://github.com/Saumya-Mishra9129), [Ibiam Chihurumnaya](https://github.com/chimosky/)
```

Comment:
Deleting the new line is shifting the Assisting Mentors title to the same line...